Books.js

```diff
@@ -0,0 +1,49 @@
+import BookClass from './bookClass.js';
+
+export default class Books {
+  constructor() {
+    this.booksList = document.getElementById('books-list');
+    this.bookTitle = document.getElementById('title');
+    this.bookAuthor = document.getElementById('author');
+    this.booksArray = [];
+    if (localStorage.getItem('BookStorage')) {
+      this.booksArray = JSON.parse(localStorage.getItem('BookStorage'));
+      this.booksDisplay();
+    }
+  }
+
+  addBook = () => {
+    const title = this.bookTitle.value;
+    const author = this.bookAuthor.value;
+    const book = new BookClass(title, author);
+    this.booksArray.push(book);
+    this.booksDisplay();
+    this.saveToLocalStorage();
+    this.bookAuthor.value = '';
+    this.bookTitle.value = '';
+  };
+
```

Comment:
Hi @joel-kalema, and thank you for your feedback. However, I don't understand how it is not working as required

Reply:
Hello @MarwanDev 
ohhh, good 😄, when we click on the `add` button, it supposes to add a book and display it, but we can't see any book. I tried to see it in my local storage but I can't also see any book

Reply:
@ joel-kalema That is weird as I'm pretty sure it works when I run it. We can even have a zoom call for verification because otherwise, I will never know how to fix it as it seems working perfectly fine from my end.